provision/config.go

```diff
@@ -56,13 +57,24 @@ type Gateway struct {
 	CfgID           string `toml:"cfg_id" json:"cfg_id"`
 }
 
+type Certs struct {
+	CAPath    string            `json:"ca_path" toml:"ca_path" `
+	CertPath  string            `json:"cert_path" toml:"cert_path" `
+	CAKeyPath string            `json:"priv_key_path" toml:"priv_key_path"`
+	DaysValid string            `json:"days_valid" toml:"days_valid"`
+	RsaBits   int               `json:"rsa_bits", toml:"rsa_bits"`
+	CA        *x509.Certificate `json:"-" toml:"-"`
+	Cert      tls.Certificate   `json:"-" toml:"-"`
```

Comment:
What this ``json:"-" toml:"-"`` means? Just curious

Reply:
it means ignore in marshall/unmarshall